### PR TITLE
Visit table - Hide Search

### PR DESCRIPTION
**Pull request by Johngtka**
Features:
- [X] `Added hiding for autocomplete and info about new visit`
- [X] `The hidding working like when you select or search patient you'll see the table`

ezgif com-video-to-gif

### CODE REVIEW

File: src/app/visit/visit.component.html [hunk 1]
```diff
@@ -66,4 +70,4 @@
     </table>
     <mat-paginator [pageSizeOptions]="[10]" aria-label="Select page">
     </mat-paginator>
-</div>
+</div>
```
Comment (macrog) [inline src/app/visit/visit.component.html:73]: your prettier doesn't format on save!

**Comment by Johngtka** [comment]
S&M commit proposal:
`feat: add hidding for search bar activated by button`

**Review by macrog — APPROVED** [review]
it works!

**Review by macrog — APPROVED** [review]
still works!

**Pull request merged**